pages/v0/formats.mdx

```diff
@@ -14,7 +14,7 @@ If you need a format that is not supported please open of a [feature request](ht
 
 Pipeless supports almost **any protocol and format** (with several codecs) as input.
 
-Supported input protocols: `v4l2` (for device webcam), `file`, `http(s)`, `rtmp`, `rtsp`, `rtp`, `tcp`, `udp`, `ftp`, ...
+Supported input protocols: `v4l2` (for device webcam), `file`, `http(s)`, `rtmp`, `rtsp`, `RTP`, `tcp`, `udp`, `FTP`, ...
```

Comment:
I would use here either all capital letters or all lowercase. I personally think sticking to lowercase is better, since we are stating out the prefix to add within the config file